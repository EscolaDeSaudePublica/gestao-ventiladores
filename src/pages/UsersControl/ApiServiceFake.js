const accessRequests = {
  labels: ['Ordem', 'Nome', 'Unidade', 'ID Saúde', 'Data de Solicitação'],
  data: [
    {
      ordem: '001',
      nome: 'José Francisco Lima',
      unidade: 'HGF',
      idSaude: '021458',
      dataSolicitacao: '08/10/2020',
    },
    {
      ordem: '002',
      nome: 'Fulana da Silva',
      unidade: 'HGF',
      idSaude: '034458',
      dataSolicitacao: '10/11/2020',
    },
    {
      ordem: '002',
      nome: 'Cicrana da Silva',
      unidade: 'HGF',
      idSaude: '034477',
      dataSolicitacao: '11/09/2020',
    },
  ],
};

const ApiService = {
  getAccessRequests: () => accessRequests,

  getAccessRequestsById: (id) => {
    accessRequests.data.filter((user) => user.idSaude === id);
  },
};

export default ApiService;
