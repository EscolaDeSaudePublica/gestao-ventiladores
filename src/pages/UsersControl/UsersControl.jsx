import React from 'react';

const UsersControl = () => (
  <><p>User Control</p></>
);

export default UsersControl;
