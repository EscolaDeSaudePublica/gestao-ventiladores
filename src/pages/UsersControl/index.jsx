import React from 'react';
import Layout from '../_layout/Layout';
import UsersControl from './UsersControl';

const UsersControlPage = () => (
  <>
    <Layout>
      <UsersControl />
    </Layout>
  </>
);

export default UsersControlPage;
