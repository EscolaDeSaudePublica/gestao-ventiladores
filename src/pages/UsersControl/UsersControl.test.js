/* eslint-disable no-undef */
/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { create } from 'react-test-renderer';
import UsersControl from './UsersControl';

describe('UsersControl component', () => {
  it('should render UsersControl component without errors', () => {
    const component = create(<UsersControl />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
