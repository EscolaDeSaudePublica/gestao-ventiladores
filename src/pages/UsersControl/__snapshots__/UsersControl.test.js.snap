// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersControl component should render UsersControl component without errors 1`] = `
<p>
  User Control
</p>
`;
